### PR TITLE
Bug 1905327: specify cpu and memory resources for initContainers

### PR DESCRIPTION
**Pull request by soltysh**
/assign @damemi

### CODE REVIEW

File: bindata/v4.1.0/kube-scheduler/pod.yaml [hunk 1]
```diff
@@ -23,6 +23,11 @@ spec:
         echo -n "."
         sleep 1
       done
+
```
Comment (damemi) [inline bindata/v4.1.0/kube-scheduler/pod.yaml:26]: are these newlines intentional?

Comment (soltysh) [reply bindata/v4.1.0/kube-scheduler/pod.yaml:26]: Yes, it's to make this more readable

**Pull request merged**